en/sensor/pmw3901.md

```diff
@@ -0,0 +1,76 @@
+# PMW3901 Flow Sensor
+
+PMW3901 is an [optical flow](../sensor/optical_flow.md) asic that computes the flow internally and provides a difference in pixels between 
+each frame over a SPI interface. It is basically a tracking sensor similar to what you would find in a computer mouse, but adapted to work between 
+80 mm and infinity. The sensor can be read at a frequency up to 100 Hz.  
+
+![Bitcraze Flow Deck](../../assets/hardware/sensors/pmw3901/bitcraze-flow.jpg)
+
+The following is a video of an S500 frame with a Pixhawk 4 and the Bitcraze PMW3901 flow deck. GPS was fully disabled in this test to see the real performance of the flow module.
+
+!!!REPLACE WITH RIGHT VIDEO
+{% youtube %}
+https://youtu.be/0Jpq6DU_HVg
+{% endyoutube %}
+
+## Boards using PMW3901
+
+The following table shows some of the boards available on the market with the pmw3901 optical flow sensor. The number interfaces, the number of sensors, the input voltage and the size are shown. The board name contains the link to where to buy the specific board.
+
```

Comment:
- Should we also list the range/height at which these can be used?
- Should we add a note that for cases with out a rangefinder one must be used as well?